tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun_test.go

```diff
@@ -68,22 +68,6 @@ func loopRunning(run *v1alpha1.Run) *v1alpha1.Run {
 	return runWithStatus
 }
 
-func requestCancel(run *v1alpha1.Run) *v1alpha1.Run {
-	runWithCancelStatus := run.DeepCopy()
-	runWithCancelStatus.Spec.Status = v1alpha1.RunSpecStatusCancelled
-	return runWithCancelStatus
-}
-
-func running(tr *v1beta1.PipelineRun) *v1beta1.PipelineRun {
```

Comment:
Just wondering why we have to remove these two unit tests? These two tests no longer valid in Tekton 0.27.0?

Reply:
they are not unit tests, they are helper functions, which are no longer used by any other functions. So cleaned them up. Thanks !

Reply:
I think they were used previously by some tests, which were removed in https://github.com/kubeflow/kfp-tekton/commit/3611c44b813ac8c491eab49258250623f1397aca#diff-9162ca2aab4cf56520cc43484b9bc56ae9232f72ee3042a2853cb235ff0d0fb1L1173 

The reason for the removal, was those tests were hanging after upgrade of tekton version i.e. 0.25. Do you think we should address that? 

Or I can undo this change here and address it in a different issue.

Reply:
thanks, I just want to make sure we didn't accidentally removed some test cases. We can merge it in this PR.